classes/Validate.php

```diff
@@ -483,7 +483,12 @@ public static function isReference($reference)
      */
     public static function isPasswd($passwd, $size = Validate::PASSWORD_LENGTH)
     {
-        return self::isPlaintextPassword($passwd, $size);
+        // Password is already a hashed no need to check
+        if (static::isHashedPassword($passwd)) {
+            return true;
+        }
+
+        return static::isPlaintextPassword($passwd, $size);
```

Comment:
@PierreRambaud Can't you extract this two methods and move it to a service ?

Reply:
It's related to Legacy context, and services are not available in the FO :thinking: